packages/paymasters/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "../../tsconfig.packages.json",
+  "include": [
+    "../paymasters/src/**/*.ts",
```

Comment:
I don't understand this. this is the `paymasters` folder. why not `./src/**/*.ts`

Reply:
Idea refactoring outsmarted itself, fixing